packages/web/src/pages/pay-and-earn-page/components/WithdrawalsTab.tsx

```diff
@@ -69,9 +68,12 @@ const DEFAULT_SORT_DIRECTION = full.GetUSDCTransactionsSortDirectionEnum.Desc
 const NoWithdrawals = () => {
   const { onOpen: openWithdrawUSDCModal } = useWithdrawUSDCModal()
   const { data: balance } = useUSDCBalance()
-  const balanceCents = formatUSDCWeiToFloorCentsNumber(
-    (balance ?? new BN(0)) as BNUSDC
-  )
+  const balanceCents =
```

Comment:
another "cents" calc that's only used once and divided by 100 later anyway :joy: